tests/v2/integration/setup/main.go

```diff
@@ -56,13 +57,21 @@ func main() {
 	}
 
 	hostURL := fmt.Sprintf("%s:8443", ipAddress.String())
-	userToken, err := token.GenerateUserToken(
-		&management.User{
+
+	var userToken *management.Token
+
+	err = kwait.Poll(500*time.Millisecond, 5*time.Minute, func() (done bool, err error) {
+		userToken, err = token.GenerateUserToken(&management.User{
 			Username: "admin",
 			Password: "admin",
-		},
-		hostURL,
-	)
+		}, hostURL)
+		if err != nil {
+			return false, nil
+		}
+
+		return true, nil
+	})
+
```

Comment:
Curious why these changes were needed? I've checked the PR description and not seeing it called out explicitly.

Reply:
This is just to reduce the errors in test..... This function sometimes errors out in the first try...so added this....it is just a copy paste from https://github.com/rancher/rancher/blob/331d18b352cfd58d5e0fc4815dffad97ca30eb49/tests/framework/extensions/pipeline/setup.go#L31